docs/source/history.md

```diff
@@ -0,0 +1,36 @@
+# History
+
+## Creation
+
+I started poliastro as a wrapper of some MATLAB and Fortran algorithms
+that I needed for a University project: having good performance was a
+must, so pure Python was not an option. As a three language project, it
+was only known to work in my computer, and I had to fight against [oct2py](https://pypi.org/project/oct2py/)
+and [f2py](https://numpy.org/doc/stable/f2py/) for long hours.
+
+Later on, I enhanced the plotting capabilities of poliastro to serve me in
+additional university tasks. I removed the MATLAB (Octave) code and kept
+only the Fortran algorithms. Finally, when numba was mature enough, I
+implemented everything in pure Python and poliastro 0.3 was born.
+
+## Future ideas
+
+These are some things that I would love to implement in poliastro to
+expand its capabilities:
+
+- 3D plotting of orbits
+- Continuous thrust maneuvers
+- Tisserand graphs
+- Porkchop plots
```

Comment:
Some of these have been implemented in the library ... I think so that we should update it with some newer features that can be added like animated plots, newer atmospheric models, & other complex orbital mechanic related things

Reply:
Oh, good catch!